src/main/java/liuni/LiUniDriver.java

```diff
@@ -1,44 +1,39 @@
 package liuni;
 
-import io.dropwizard.Configuration;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import liuni.health.LiUniHealthCheck;
 import liuni.resources.LiUniResource;
 import io.dropwizard.Application;
 import io.dropwizard.setup.Environment;
-import twitter4j.TwitterFactory;
+import twitter4j.Twitter;
 
-import java.io.BufferedWriter;
-import java.io.FileWriter;
-import java.io.IOException;
-
-public class LiUniDriver extends Application<Configuration> {
+public class LiUniDriver extends Application<TwitterConfig> {
     public static void main(String args[]) {
-        setupTwitter();
         try {
             new LiUniDriver().run(args);
         }
         catch (Exception e) {
             e.printStackTrace();
         }
     }
+
     @Override
-    public void run(Configuration config, Environment env) {
+    public void run(TwitterConfig config, Environment env) {
+        Twitter twitter = config.createTwitterConfig();
```

Comment:
do you need to be configuration twitter at this level?

Reply:
Resolved